deploy/mock-identity-system/install.sh

```
@@ -59,14 +59,88 @@ function installing_mock-identity-system() {
+  HSM_HELM_ARGS=''
+  if [[ $enable_volume == 'true' ]]; then
+    default_volume_size=100M
+    read -p "Provide the size for volume [ default : 100M ]" volume_size
+    volume_size=${volume_size:-$default_volume_size}
+
+    default_volume_mount_path='/home/mosip/config/'
+    read -p "Provide the mount path for volume [ default : '/home/mosip/config/' ] : " volume_mount_path
+    volume_mount_path=${volume_mount_path:-$default_volume_mount_path}
+
+    default_keystore_pass='1234'
+    read -p "Provide the PKCS12 keystore password [ default : 1234 ] : " keystore_pass
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)